core/capabilities/webapi/target/target.go

```diff
@@ -57,6 +58,12 @@ func (c *Capability) Start(ctx context.Context) error {
 }
 
 func (c *Capability) Close() error {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
```

Comment:
can you expand on why the timeout of 1s on all the remove calls?  I guess cross loop boundary it would not be unrealistic  for this to be breached occasionally